app.py

```diff
@@ -0,0 +1,103 @@
+from main.mains import Amity
+import argparse
+
+x = Amity()
```

Comment:
**Code Quality: Critical**

Use better variable names. `x` has no meaning in the context of your program.